src/pretix/control/views/event.py

```diff
@@ -195,6 +196,7 @@ def form_valid(self, form):
         self.save_meta()
         self.save_item_meta_property_formset(self.object)
         self.save_confirm_texts_formset(self.object)
+        self.save_footer_links_formset(self.object)
```

Comment:
Why not just a `self.footer_links_formset.save()`? I guess it is for consistency with the other `save_*` above?

Reply:
Yes, in case we need future logic and to keep it consistent